src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx

```diff
@@ -500,7 +500,7 @@ class FormField extends React.Component {
 
     const observedProps = propsToObserver
       .filter(p => typeof this.props[p] === 'function')
-      .map(p => [p, () => this.props[p](model)]);
+      .map(p => [p, () => this.props[p]({...this.props, model})]);
```

Comment:
@EvanPurkhiser do you remember other places where this is used? I couldn't find instances (besides `<ChoiceMapperField>`), including getsentry.